aiohttp/client_reqrep.py

```diff
@@ -35,11 +35,14 @@
 
 
 class ClientRequest:
-
-    GET_METHODS = {hdrs.METH_GET, hdrs.METH_HEAD, hdrs.METH_OPTIONS}
+    GET_METHODS = {
+        hdrs.METH_GET,
+        hdrs.METH_HEAD,
+        hdrs.METH_OPTIONS,
+        hdrs.METH_TRACE,
```

Comment:
> Should I add raise ValueError in constructor in these requests ?

Let's open another issue for that. If we can help users to not produce invalid requests that would be good.